### PR TITLE
Fix: broken rendering in create new scaler markdown file

### PR DESCRIPTION
**Pull request by dttung2905**
Hi team,

This MR fixes the broken rendering in markdown file to create a new scaler. Let me know if I need to update CHANGELOG file if it is for Document changes
### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

### CODE REVIEW

**Review by zroubalik — COMMENTED** [review]
Thanks, good stuff, could you please also incorporate following changes, which were omitted in the last update?

File: CREATE-NEW-SCALER.md [hunk 1]
```diff
@@ -110,4 +110,4 @@ Scalers are created and cached until the ScaledObject is modified, or `.IsActive
 The scaler code is embedded into the two separate binaries comprising KEDA, the operator and the custom metrics server component. The metrics server must be occasionally rebuilt published and deployed to k8s for it to have the same code as your operator.
 
 GetMetricSpecForScaling() is executed by the operator for the purposes of scaling out and in to 0 replicas.
-GetMetrics() is executed by the custom metrics server in response to a calls against the external metrics api, whether by the HPA loop or by curl
+GetMetrics() is executed by the custom metrics server in response to a calls against the external metrics api, whether by the HPA loop or by curl.
```
[the remark this answers is not in the document]
Comment (dttung2905) [reply CREATE-NEW-SCALER.md]: Thanks for the quick review!. I have updated it :pray:

**Review by zroubalik — APPROVED** [review]
LGTM, thanks!

**Comment by dttung2905** [comment]
Thanks @tomkerkhove ! It should be fixed now.

**Review by zroubalik — APPROVED** [review]
thanks!

**Pull request merged**